pkgs/games/0ad/game.nix

```diff
@@ -1,30 +1,115 @@
-{ stdenv, lib, perl, fetchurl, python2
-, pkg-config, spidermonkey_38, boost, icu, libxml2, libpng, libsodium
-, libjpeg, zlib, curl, libogg, libvorbis, enet, miniupnpc
-, openal, libGLU, libGL, xorgproto, libX11, libXcursor, nspr, SDL2
-, gloox, nvidia-texture-tools
-, withEditor ? true, wxGTK ? null
+{stdenv
+,cargo
```

Comment:
Commonly we add a space after the comma whenever we use this style for dependencies.

e.g.:

```
{ stdenv
, cargo
, lib
, ...
}
```